articles/Getting-started-with-windows-forms-using-c-#/index.md

```diff
@@ -0,0 +1,176 @@
+**Windows forms** allow the programmers to create a graphical user interface (GUI) that allows the interaction with the program visually. According to [Microsoft](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/overview/?view=netdesktop-5.0), It provides one of the most productive ways to create desktop apps based on the visual designer provided in Visual Studio. Functionality such as drag-and-drop placement of visual controls makes it easy to build desktop apps.
+
+With *windows forms*, you can make basic applications like a calculator to high-level applications such as a library database management application that requires a connection to the database from the application.
+### Prerequisites
+1. A basic understanding of c# programming language.
+2. Visual studio installed on your system.
+
+If you don't have Visual Studio installed on your computer. you can download it from [Here](https://visualstudio.microsoft.com/downloads/).
```

Comment:
Instead of just providing the link for the Visual Studio code, can you provide an article or documentation page that explains how to set up the C# environment in Visual Studio.